### ISSUE TITLE
Support for Windows and macOS?

### ISSUE THREAD

**Issue by pm64** [JinShil/BlazorWebView#8]
I hope this is not off-base, but I wonder if you'd consider adding support for Windows and/or macOS? Currently we have no single BlazorWebView solution working across all 3 platforms.

**Comment by JinShil** [comment]
Your question is not off-base and perfectly understandable.  At the moment, however, the position I'm taking is that you should split your program into multiple projects:

1.  A library project that contains your Blazor application.
2. A Blazor project for Linux that contains a reference the BlazorWebView project in this repository as well as a reference to your application library project created in 1.
3. A Windows WPF/Winforms Blazor hybrid project that has a reference to the application library project created in 1.
4. A MacOS Blazor hybrid project that has a reference to the application library projected created in 1.

So basically, you'd pull your entire application out into a library.  Then create a separate project for each platform, where each of those projects references your application library.  The Linux project would utilize the BlazorWebView in this repository.

I hope that makes sense.  If I can find the time, I might create an example in this repository demonstrating that structure as a reference.

**Comment by pm64** [comment]
Thanks @JinShil for the thoughtful response. I'll experiment with this project structure and report back with any findings.
